modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -4856,6 +4856,24 @@ Transaction manager's synchronization timeout. Maximum time to wait for internal
 
 ---
 
+=== tombstone_retention_ms
+
+The retention time for tombstone records in a compacted topic. Cannot be enabled at the same time as any of `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, or `cloud_storage_enable_remote_write`.
+
+*Unit:* milliseconds
+
+*Requires restart:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `null`
+
```

Comment:
We could also add the main doc as a related topic? The xref will be `xref:manage:cluster-maintenance/compaction-settings.adoc#tombstone-record-removal[Tombstone record removal]`